packages/server/src/main/resources/application-local.properties

```diff
@@ -41,7 +41,7 @@ spicedb.target=localhost:50051
 ########################################################################################################################
 aws-access-key-id=admin
 aws-secret-access-key=admin123
-aws-url=http://minio:9000
+aws-url=http://localhost:9000
```

Comment:
You'll need to add `minio` to your `/etc/hosts/` file for this. As-is `localhost` may have issues working with docker, etc

Reply:
local is used within docker?  we should have an application-docker.properties, but ok